plugins/catalog-backend-module-aws/api-report.md

```diff
@@ -43,4 +54,36 @@ export class AwsS3DiscoveryProcessor implements CatalogProcessor {
     parser: CatalogProcessorParser,
   ): Promise<boolean>;
 }
+
+// @public
+export class AwsS3EntityProvider implements EntityProvider {
+  constructor(
+    config: AwsS3Config,
+    integration: AwsS3Integration,
+    logger: Logger,
+    schedule: TaskRunner,
+  );
+  // (undocumented)
+  connect(connection: EntityProviderConnection): Promise<void>;
+  // (undocumented)
+  createLocationSpec(key: string): LocationSpec;
+  // (undocumented)
+  createObjectUrl(key: string): string;
```

Comment:
I think most of these can be made private right?

Reply:
Yes. I was wondering which of them need to be private and which of them to keep open for overriding, etc. I guess `protected` does not really make a big difference for what your after either.

I can mark them private and however needs anything could make a PR to open it up more again :)

Reply:
done